parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -210,23 +210,31 @@ configureCNIIPTables() {
     fi
 }
 
+disableSystemdResolved() {
+    echo "Ignoring systemd-resolved query service but using its resolv.conf file"
+    echo "This is the simplest approach to workaround resolved issues without completely uninstalling it"
+    [ -f /run/systemd/resolve/resolv.conf ] && sudo ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
+    ls -ltr /etc/resolv.conf
+    cat /etc/resolv.conf
+}
+
 disable1804SystemdResolved() {
     ls -ltr /etc/resolv.conf
     cat /etc/resolv.conf
     {{- if Disable1804SystemdResolved}}
     UBUNTU_RELEASE=$(lsb_release -r -s)
     if [[ ${UBUNTU_RELEASE} == "18.04" ]]; then
-        echo "Ingorings systemd-resolved query service but using its resolv.conf file"
-        echo "This is the simplest approach to workaround resolved issues without completely uninstall it"
-        [ -f /run/systemd/resolve/resolv.conf ] && sudo ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
-        ls -ltr /etc/resolv.conf
-        cat /etc/resolv.conf
+        disableSystemdResolved
     fi
     {{- else}}
     echo "Disable1804SystemdResolved is false. Skipping."
     {{- end}}
 }
 
+disableSystemdIptables() {
+    systemctlDisableAndStop iptables
```

Comment:
systemctlDisableAndStop iptables || exit ERR_DISBALE_IPTABLES

Reply:
also let's bake this in VHD instead of doing it here in cse.

Reply:
Moved both the iptables and systemdresolved cache fix into the VHD build for Mariner instead of CSE.